### ISSUE TITLE
line 26 of update.php invalid

### ISSUE THREAD

**Issue by m-hume** [kbabioch/php-ddns#1]
line 26 is not valid php
```$key = 'hmac-sha256:name KEY')```

could we get a description on what this is supposed to be?
cheers

**Comment by bigon** [comment]
key is supposed to be

`$key = 'hmac-sha256:name KEY';` (there is a typo at the end) where name is the name of the key defined in bind configuration and KEY the key itself

**Comment by kbabioch** [comment]
Yes, there is indeed a typo (missing semi-colon). Thank you for noticing.

This is basically a variable that contains the secret that is used to have a secure communication between this script and the nameserver.

In reality it will look something like this:

```php
$key = 'hmac-sha256:ddns-update XXXXXXXXXX';
```

You can generate such a key using the `tsig-keygen` command:

```shell
[root@ns1 ~]# tsig-keygen -a hmac-sha256 example.com
key "example.com" {
        algorithm hmac-sha256;
        secret "RozyZlWfIa3arEJDG5mhINyW1mPyCR59ydvAZbye3O4=";
};
```

I guess this needs better / further documentation. Will update it eventually, leaving this issue open in the mean time.
